README.md

```diff
@@ -153,47 +159,59 @@ For using this tool in a distributed tracing scenario, where there is a parent t
 
 For further reference on environment variables in the OpenTelemetry SDK, please read the [official specification](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/)
 
-## OpenTelemetry Attributes
-This tool is going to parse the XML report produced by jUnit, or any other tool converting to that format, adding different attributes, separated by different categories:
+## Traces
 
-- Test metrics attributes
-- Ownership attributes
+Traces are sent to the OpenTelemetry collector, representing the test execution. Each run of the tool will create a root trace that contains spans for each suite. Each suite will contain spans for each test case.
 
-### Metrics and Traces
-The following attributes are added as metrics and/or traces.
+In addition to the common attributes, the tool will add the following attributes to the trace document representing the test execution:
```

Comment:
Nit: seems a little weird to have the `:` and then start a new section at the same heading level as this.

It's not clear to me if the metrics are "attributes". Possibly add another `#` to the sections in the scope of "following attributes"?

Reply:
Great call out. That is vestigial. I've removed that piece and added more clarification.